app/src/main/java/turtler/voyageur/adapters/ImageGridAdapter.java

```diff
@@ -0,0 +1,62 @@
+package turtler.voyageur.adapters;
+
+import android.app.Activity;
+import android.content.Context;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ArrayAdapter;
+import android.widget.BaseAdapter;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.squareup.picasso.Picasso;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import turtler.voyageur.R;
+import turtler.voyageur.models.Image;
+import turtler.voyageur.models.ImageItem;
+
+/**
+ * Created by skulkarni on 9/13/16.
+ */
+public class ImageGridAdapter extends ArrayAdapter {
```

Comment:
can we just create an imageadapter based on the Parse Image class? It'll be easier in the future to connect an image to an event or a trip or something